Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-da.sh
+# Description:
+#    Validate the installation and uninstallation of the Dependency Agent.
+#    Validate the enable and disable of the Dependency Agent.
+#######################################################################
+
+set -e
+set -x
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Get distro information
+GetDistro
+
+dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency-agent /etc/opt/microsoft/dependency-agent /lib/microsoft-dependency-agent /etc/init.d/microsoft-dependency-agent)
+supported_distro_list=(redhat centos suse debian ubuntu)
+
+readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+
+function test_case_cleanup(){
+    /opt/microsoft/dependency-agent/uninstall
```

Comment:
Test for existence of the file before attempting to execute. Otherwise, it will cause a non-zero exit status, and because of the `set -e` above will cause the shell to terminate immediately.

Reply:
Done